mrtarget/plugins/gene/tractability.py

```diff
@@ -75,15 +75,16 @@ def build_json(self, filename):
         ab_bucket_list = [1, 2, 3, 4, 5, 6, 7, 8, 9]
 
         with URLZSource(filename).open() as r_file:
-            for i, el in enumerate(csv.DictReader(r_file,
+            # Field names are provided so that spaces in column names do not cause problems, skip first row though
+            for i, row in enumerate(csv.DictReader(r_file,
                                                   fieldnames=tractability_columns,
-                                                    dialect='excel-tab'), start=1):
+                                                    dialect='excel-tab'), start=2):
```

Comment:
@MichaelaEBI why do you want to start counting at 2? enumerate is just a izip with a natural generator (1, next(k)) (2, next(k)) .... in the case you may just want to drop few lines use consume with the r_file from more_itertools package as this example
```python
In [1]: import more_itertools

In [2]: list_of_10 = iter(xrange(10))

In [3]: more_itertools.consume(list_of_10, 2)

In [4]: print(list(list_of_10))
[2, 3, 4, 5, 6, 7, 8, 9]
```

Reply:
Thank you Miguel! I am checking with Chris from ChEMBL whether he would be happy to adjust the tractability file header to not contain spaces. If he is happy to do that, then we do not need to skip a row. I will fix the code as soon as I know what is going to happen.

Reply:
Chris is happy to amend the header for future tractability files so that we can use the header in the file. The code has been adjusted to use the first line in the file as the header.